ClosedXML/Excel/Drawings/XLPicture.cs

```diff
@@ -14,39 +14,43 @@ namespace ClosedXML.Excel.Drawings
     internal class XLPicture : IXLPicture
     {
         private const String InvalidNameChars = @":\/?*[]";
-        private static IDictionary<XLPictureFormat, ImageFormat> FormatMap;
+        private static IDictionary<XLPictureFormat, SKEncodedImageFormat> FormatMap;
         private Int32 height;
         private Int32 id;
         private String name = string.Empty;
         private Int32 width;
 
         static XLPicture()
         {
-            var properties = typeof(ImageFormat).GetProperties(BindingFlags.Static | BindingFlags.Public);
-            FormatMap = Enum.GetValues(typeof(XLPictureFormat))
-                .Cast<XLPictureFormat>()
-                .Where(pf => properties.Any(pi => pi.Name.Equals(pf.ToString(), StringComparison.OrdinalIgnoreCase)))
-                .ToDictionary(
-                    pf => pf,
-                    pf => properties.Single(pi => pi.Name.Equals(pf.ToString(), StringComparison.OrdinalIgnoreCase)).GetValue(null, null) as ImageFormat
-                );
+            var properties = typeof(SKEncodedImageFormat).GetProperties(BindingFlags.Static | BindingFlags.Public);
```

Comment:
it seems like the properties-Variable is not used => remove

Reply:
It looks to me like this might have had something to do with flags enums (where there can be "Properties" with the same values) in System.Drawing. I honestly did not check. I suppose SKEncodedImageFormat is not a flags enum?

Reply:
Addressed: not used variable dropped